Buildings/Resources/Scripts/Dymola/Examples/HydronicHeating/TwoRoomsWithStorage.mos

```diff
@@ -13,7 +13,7 @@ createPlot(id = 5,
  bottomTitleType = 1);
 createPlot(id = 5,
  position = {10, 30, 719, 175},
- y = {"tan.vol[1].T", "tan.vol[2].T", "tan.vol[3].T", "tan.vol[4].T", "tan.vol[5].T", "boi.sta_b.T", "temSup.T", "temRet.T"},
```

Comment:
Please plot `boi.T` so that we have boiler temperature in the plots. This is the volume's temperature and hence does not cause problems.